test/ExecutionFramework.cpp

```diff
@@ -170,6 +171,33 @@ void ExecutionFramework::sendMessage(bytes const& _data, bool _isCreation, u256
 		m_transactionSuccessful = (m_gas != m_gasUsed);
 }
 
+void ExecutionFramework::waitForTransaction(std::string const& _txHash) const
+{
+	int polls = 0;
+	while(true)
+	{
+		polls++;
+		auto pendingBlock = m_rpc.eth_getBlockByNumber("pending", false);
+
+		bool isEmpty = pendingBlock["transactions"].empty();
+
+		if (isEmpty && polls < 3000)
+		{
```

Comment:
I think if we bombard aleth with requests on the rpc interface, this will even worsen the situation. Perhaps add a microsleep here?